### PR TITLE
Version 4.0.0

### PR DESCRIPTION
**Pull request by smolentzov-stripe**

### CODE REVIEW

File: src/Taxjar.sln
Comment (saville-stripe) [inline src/Taxjar.sln:1]: Any reason this is showing total diff? Seems like most of it is actually the same 🤔

File: build/TaxJar.nuspec [hunk 1]
```diff
@@ -12,20 +12,21 @@
     <iconUrl>https://github.com/taxjar/taxjar.net/raw/master/build/taxjar-icon.png</iconUrl>
     <description>TaxJar makes sales tax filing easier for online sellers and merchants. See local jurisdictional tax reports, get payment reminders, and more. You can use our API to access TaxJar API endpoints, which can get information on sales tax rates, categories or upload transactions.</description>
     <releaseNotes>
-- Update RestSharp to version 106.12.0
+- Update RestSharp to 108.0.3
```
Comment (saville-stripe) [inline build/TaxJar.nuspec:15]: Should these be indented or is it a `nuspec` thing?

Comment (smolentzov-stripe) [reply build/TaxJar.nuspec:15]: Good question, let me investigate!

Comment (smolentzov-stripe) [reply build/TaxJar.nuspec:15]: @saville-stripe I'm not entirely sure on the "why" still, but no indentation seems to have been the standard here, looking through history. I found an example of a previous version with multiple lines and it was the same format: https://github.com/taxjar/taxjar.net/blob/14e329083398873961fa3b36e459bf7f8bf3f688/build/TaxJar.nuspec

Also this stackoverflow is interesting. I guess whitespace doesn't really matter for XML in general, so perhaps it's just a personal preference? https://stackoverflow.com/questions/12526438/importance-of-indenting-xml-file

File: src/Taxjar.sln
Comment (smolentzov-stripe) [reply src/Taxjar.sln:1]: I was also confused by this... the only thing that's changed is the version number. Maybe GH does something weird with this file format?

Comment (saville-stripe) [reply src/Taxjar.sln:1]: It's probably not a huge deal, curious tho!

**Review by saville-stripe — APPROVED** [review]
Seem right to me!

**Pull request merged**